libr/util/ctype.c

```diff
@@ -423,6 +423,13 @@ R_API char *r_type_format(Sdb *TDB, const char *t) {
 		fmt = r_str_append (fmt, vars);
 		free (vars);
 		return fmt;
+	} else if (!strcmp (kind, "typedef")) {
+		snprintf (var2, sizeof (var2), "typedef.%s", t);
+		const char *type = sdb_const_get (TDB, var2, NULL);
+		// only supports struct atm
+		if (type && !strncmp (type, "struct ", 7)) {
+			return r_type_format (TDB, type + 7);
```

Comment:
isnt type +7 the same as just t?

Reply:
only for a specific case that should be rare but it causes infinite recursion so fixed in cd4f7164169f640f79b066eec0994379af4aad88